### DISCUSSION TITLE
How to configure rewrites with a domain ports

### DISCUSSION THREAD

**Discussion by lancelot-lewis** [vercel/vercel#4489] [Help] [answered]
How to configure `rewrites` to a url with custom port, like this:
```
"rewrites": [
  { 
    "source": "/api/(.*)",
    "destination": "http://v3.dagoovis.com/api/$1"
   }
]
```

**Comment by styfle** [comment, marked as answer]
That looks correct to me. Is this not working?

**Reply by lancelot-lewis** [reply]
Right, it's will not work, confirmed by vercel
![image](https://user-images.githubusercontent.com/8749615/82964233-89736100-9ff7-11ea-9a81-575a87625227.png)
